components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenMgtDAO.java

```diff
@@ -1741,6 +1741,7 @@ public void revokeToken(String tokenId, String userId) throws IdentityOAuth2Exce
         String userDomain = getSanitizedUserStoreDomain(authzUser.getUserStoreDomain());
 
         try {
+            Connection connection = IdentityDatabaseUtil.getDBConnection();
```

Comment:
Is this connection needed? Seems there is no usages.

Reply:
Connection is used to execute the SQL query below